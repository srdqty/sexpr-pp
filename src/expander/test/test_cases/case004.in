(lambda (x)
  (let-syntax (
    (n  (lambda (stx)
          ; expand (n e) to (lambda (x) (+ e x))
          (mk-stx
            (list (syntax lambda) (syntax (x))
              (mk-stx
                (list (syntax +) (car (cdr (stx-e stx))) (syntax x))
                stx))
            stx)))
    )
    (n 1)))
