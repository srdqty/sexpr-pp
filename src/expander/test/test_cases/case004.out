>> (lambda (x0) (begin (lambda (x1) (+ 1 x1))))
>> 
